src/main/java/com/_up/megastore/controllers/interfaces/IOrderController.java

```diff
@@ -3,16 +3,23 @@
 import com._up.megastore.controllers.requests.CreateOrderRequest;
 import com._up.megastore.controllers.responses.OrderResponse;
 import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.ResponseStatus;
 
+import java.util.UUID;
+
 @RequestMapping("/api/v1/orders")
 public interface IOrderController {
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
     OrderResponse saveOrder(@RequestBody CreateOrderRequest createOrderRequest);
 
+    @PostMapping("/{orderId}/finish")
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
Deberia ser un OK, ya que no se crea una orden sino que se cambia su estado luego de cometer una accion en particular


```suggestion
    @ResponseStatus(HttpStatus.OK)
```